packages/stores/src/storeProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropsWithChildren, useEffect, useMemo } from 'react';
+import StoreContext from './storeContext';
+import { ExchangeRatesStore } from './stores';
+import { ExchangeRatesProvider } from './providers';
+import type { TRootStore, TStores } from '../types';
+
+const StoreProvider = ({ children, store }: PropsWithChildren<{ store: TRootStore }>) => {
+    const memoizedValue: TStores = useMemo(
+        () => ({
+            ...store,
+            exchange_rates: new ExchangeRatesStore(),
+        }),
+        [store]
```

Comment:
this will update on each change on store?
is it a good parameter to use in dep array?

Reply:
no it won't